src/config.ts

```diff
@@ -342,6 +342,14 @@ export let ENABLE_ANONYMOUS_USER : boolean = false;
 */
 export let ENABLE_DEVELOPER_PROGRAM : boolean = false;
 
+/**
+  Enable developer backend.
+
+  User dedicated pod to host trusted developer backend. This will increase
+  startup time of the developer as a new pod will be spin up from a cold start.
```

Comment:
We should also note that this is necessary for security if ENABLE_DEVELOPER_PROGRAM is set.